index.html

```diff
@@ -38,6 +34,149 @@ <h5 class="btn-connect">LET’S CONNECT</h5>
             </ul>
         </section>
     </header>
+    <!-- cord section start from here -->
+    <section class="card-section">
+        <div class="card">
+            <div class="card-heading">
+                <img src="images/card.png" alt="card">
+            </div>
+            <div class="card-body">
+                <h1 class="card-title">Tonic</h1>
+                <div class="card-author">
+                    <h5 class="heading5">CANOPY</h5>
+                    <div class="dot"></div>
+                    <p>Back End Dev</p>
+                    <div class="dot"></div>
+                    <p>2015</p>
+                </div>
+                <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.</p>
+            </div>
+            <div class="card-languages">
+                <p class="language-item ">html</p>
+                <p class="language-item ">css</p>
+                <p class="language-item ">javascript</p>
+            </div>
+            <div class="btn-card-one">
+                See Project
+            </div>
```

Comment:
- [ ] Please use the button for your (e.g., "See project") link `a` or `button` tag, and  follow the button styling: Active, Hover, and Pressed as defined on the template :+1:

![Screenshot from 2022-08-20 21-36-02](https://user-images.githubusercontent.com/6615394/185766765-8da9017d-3f95-4db1-b3ba-bc226f788c06.png)